api/core.py

```diff
@@ -416,7 +430,11 @@ def _archivestream(self, ticket):
         stream = cStringIO.StringIO()
         with tarfile.open(mode='w|', fileobj=stream) as archive:
             for filepath, arcpath, _ in ticket['target']:
-                yield archive.gettarinfo(filepath, arcpath).tobuf()
+                try:
+                    yield archive.gettarinfo(filepath, arcpath).tobuf()
+                except Exception as e:
```

Comment:
As stated before, I agree with your sentiment, @kofalt. This logic has however been in the code for a long time, in [one form or another](https://github.com/scitran/api/blob/master/api/core.py#L160). I opened a new issue (#119) for further discussion of this topic.